material-overrides/main.html

```diff
@@ -47,6 +47,7 @@
   <style data-emotion="mantine" data-s=""></style>
   <script defer
     src="https://widget.kapa.ai/kapa-widget.bundle.js"
+    data-user-analytics-cookie-enabled="false"
     data-website-id="3fbec0cf-5f4b-4f78-9cff-4ebf071c3bd3"
```

Comment:
Nitpick here, could we please do it in the exact same order as the docs state -> https://docs.kapa.ai/integrations/website-widget/user-tracking#disabling-user-tracking? It would be easier to debug bugs in the future if any.

```suggestion
    data-user-analytics-cookie-enabled="false"
    data-website-id="3fbec0cf-5f4b-4f78-9cff-4ebf071c3bd3"
```